resources-domain/resources-service/src/resource/resourceRoutesV0.ts

```diff
@@ -48,6 +48,17 @@ const resourceRoutes = () => {
     res.json(referrableResources);
   });
 
+  router.get('/suggest', async (req, res) => {
+    const { size, prefix } = req.query;
```

Comment:
We will probably need a 'category' of some sort in the future, either as an input parameter or  - some of the discussions around autocomplete are around pulling suggestions from a number of different autocomplete sources, and then showing them in the UI in their categories

But there's no requirement for that now so no need to add it in this PR, just wanted to mention it for context